src/.bazelrc

```diff
@@ -1,5 +1,5 @@
 # Use Our Remote Cache To Avoid Rebuilding Files Already Built By CI
-build --remote_http_cache=https://storage.googleapis.com/thunderbots_bazel_cache_us_multi_region
+# build --remote_http_cache=https://storage.googleapis.com/thunderbots_bazel_cache_us_multi_region
```

Comment:
Is this intentional?

Reply:
The remote cache actually got removed in #1062 so this will be gone entirely once conflicts are resolved.